README.md

```diff
@@ -1,2 +1,158 @@
-# kafka-connect-drop-field-smt
-A Kafka Connect SMT to drop (nested) fields
+[![Build Status](https://dev.azure.com/bakdata/public/_apis/build/status/bakdata.kafka-connect-plugins?branchName=main)](https://dev.azure.com/bakdata/public/_build/latest?definitionId=35&branchName=main)
+[![Sonarcloud status](https://sonarcloud.io/api/project_badges/measure?project=com.bakdata.kafka%3Akafka-connect-plugins&metric=alert_status)](https://sonarcloud.io/project/overview?id=com.bakdata.kafka:kafka-connect-plugins)
+[![Code coverage](https://sonarcloud.io/api/project_badges/measure?project=com.bakdata.kafka%3Akafka-connect-plugins&metric=coverage)](https://sonarcloud.io/project/overview?id=com.bakdata.kafka:kafka-connect-plugins)
+[![Maven](https://img.shields.io/maven-central/v/com.bakdata.kafka-connect-plugins/kafka-connect-plugins.svg)](https://search.maven.org/search?q=g:com.bakdata.kafka-connect-plugins%20AND%20a:kafka-connect-plugins&core=gav)
+
+# Kafka Connect plugins
+
+A collection of Kafka Connect plugins.
+
+## Single Message Transforms (SMTs)
+
+### Convert
+
+#### Description
+
+Converts a byte record using the given converter class.
+The [MirrorMaker](https://github.com/apache/kafka/blob/trunk/connect/mirror/README.md)
+connector uses byte array records.
+To apply other SMTs to these records,
+we need to convert them to the appropriate format first.
+
+Use the concrete transformation type designed for the record key (`com.bakdata.kafka.Convert$Key`)
+or value (`com.bakdata.kafka.Convert$Value`).
+
+#### Example
+
+This configuration snippet shows how to use `Convert`.
+It converts the value to a string schema.
+
+```yaml
+"transforms": "convert",
+"transforms.convert.type": "com.bakdata.kafka.Convert$Value",
+"transforms.convert.converter": "org.apache.kafka.connect.storage.StringConverter"
+```
+
+#### Properties
+
+| Name        | Description                  | Type  | Default                  | Valid Values                                                                                                                                    | Importance |
+|-------------|------------------------------|-------|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|------------|
+| `converter` | Converter to apply to input. | class | ByteArrayConverter.class | All classes that implement the [Kafka Converter interface](https://kafka.apache.org/34/javadoc/org/apache/kafka/connect/storage/Converter.html) | high       |
```

Comment:
Not required, but I think it would be a bit more consistent:
```suggestion
| `converter` | Converter to apply to input. | class | `ByteArrayConverter.class` | All classes that implement the [Kafka Converter interface](https://kafka.apache.org/34/javadoc/org/apache/kafka/connect/storage/Converter.html) | high       |
```